includes/class-wc-taxjar-api-calculation.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class WC_Taxjar_API_Calculation {
+
+	/**
+	 * @var WC_Taxjar_Integration
+	 */
+	public $taxjar_integration;
+
+	/**
+	 * WC_Taxjar_API_Calculation constructor.
+	 */
+	public function __construct( $integration ) {
+		$this->taxjar_integration = $integration;
+
+		if ( $this->is_api_calculation_enabled() ) {
+			// Calculate tax during creation and update of order through WooCommerce REST API
+			add_filter( 'woocommerce_rest_pre_insert_shop_order_object', array( $this, 'calculate_api_order_tax'), 20, 3);
+		}
+	}
+
+	/**
+	 * Determine is tax calculation on API orders is enabled
+	 *
+	 * @return bool
+	 */
+	public function is_api_calculation_enabled() {
+		return isset( $this->taxjar_integration->settings['api_calcs_enabled'] ) && 'yes' == $this->taxjar_integration->settings['api_calcs_enabled'];
```

Comment:
Just curious, why is this value `yes` and not `true`?

Reply:
That's just a peculiarity of the WooCommerce settings API our plugin utilizes.